### PR TITLE
Review date - Refactor guidance and examples on the service manual

### PR DESCRIPTION
**Pull request by AdamChrimes**
## Description

remove the code for the review date component from the frontend library and use the helper classes to achieve the same thing.

Start refactoring the component into a pattern.

### Related issue
https://github.com/nhsuk/nhsuk-frontend/pull/710

### Depends on
https://github.com/nhsuk/nhsuk-service-manual/issues/989

## Checklist

- [x] Tested against the [NHS.UK frontend testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [x] Code follows the [NHS.UK frontend coding standards](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/coding-standards.md)
- [x] Version number has been increased in `package.json` (using [SEMVER](https://semver.org/))
- [x] CHANGELOG entry
- [x] [Whats new page](https://service-manual.nhs.uk/whats-new) entry
- [x] Page updated date

### CODE REVIEW

File: app/views/cookie-confirmation.njk [hunk 1]
```diff
@@ -32,8 +32,8 @@
 
     </div>
 
-    <div class="nhsuk-review-date">
-      <p class="nhsuk-body-s">Updated: January 2020</p>
+    <div class="nhsuk-u-margin-top-7">
```
Comment (AdamChrimes) [inline app/views/cookie-confirmation.njk:35]: should this just be

```
<p class="nhsuk-body-s nhsuk-u-margin-top-7 nhsuk-u-secondary-text-color">Updated: January 2020</p>
```

**Comment by AdamChrimes** [comment]
Needs CHANGELOG

**Comment by davidhunter08** [comment]
Blocked until we finish https://github.com/nhsuk/nhsuk-service-manual/issues/989

**Review by sarawilcox — APPROVED** [review]
Looking good.

**Pull request merged**